### PR TITLE
feat(ticket # 146) : work on common form title comonent

### PR DESCRIPTION
**Pull request by nadirhussain786**
## Description

This PR contains a common title for all forms and this title is designed according to themes styles .

### Resource

image

### CODE REVIEW

File: src/common/MemberSelector/MemberSelector.tsx [hunk 1]
```diff
@@ -88,7 +88,7 @@ const MemberSelector = ({ control, name }: SelectFieldProps) => {
           <FormControl>
             <Select
               {...field}
-              value={field.value}
+              value={loginUser.role === "member"  ? loginUser.name : field.value}
```
Comment (muhammad-bilal-ali-khokhar) [inline src/common/MemberSelector/MemberSelector.tsx:91]: @nadirhussain786 use enum

File: src/common/FormTitle/FormTitle.tsx
Comment (muhammad-bilal-ali-khokhar) [inline src/common/FormTitle/FormTitle.tsx:1]: nice

File: src/app/leaves/add/components/LeaveRequestFormBuilder.tsx [hunk 2]
```diff
@@ -94,9 +106,7 @@ export const LeaveRequestFormBuilder = () => {
 
   return (
     <FormWrapper>
-      <h1 className="text-sm text-sm font-semibold text-gray-800 my-4">
-        Request Leave
-      </h1>
+      <FormTitle title="Request Leave" />
```
Comment (muhammad-bilal-ali-khokhar) [inline src/app/leaves/add/components/LeaveRequestFormBuilder.tsx:109]: cool

**Review by muhammad-bilal-ali-khokhar — APPROVED** [review]

File: src/common/FormTitle/FormTitle.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export const FormTitle = ({ title }: { title: string }) => {
+  return (
+    <h1 className="text-sm text-sm font-semibold text-gray-800 my-4">
```
Comment (Mubashar1009) [inline src/common/FormTitle/FormTitle.tsx:5]: two time same class is using text-sm

**Review by ghulamhafeez — APPROVED** [review]

File: src/app/leaves/add/components/LeaveRequestFormBuilder.tsx [hunk 4]
```diff
@@ -57,15 +60,24 @@ const LEAVE_REQUEST_TYPES = [
...
+  updated_at: new Date().toISOString(),
+  deleted_at: null,
 };
+
 export const LeaveRequestFormBuilder = () => {
   const form = useForm<z.infer<typeof leaveRequestSchema>>({
     resolver: zodResolver(leaveRequestSchema),
     defaultValues: {
-      memberId: user.role === UserRole.Member ? user.id.toString() : "",
+      memberId: loginUser.role === UserRole.Member ? loginUser.id.toString() : "",
```
Comment (Monfernape) [inline src/app/leaves/add/components/LeaveRequestFormBuilder.tsx:80]: @nadirhussain786 we need to learn how to not get our PRs out of context. This PR was only for form title but we expanded to other areas.

Comment (nadirhussain786) [reply src/app/leaves/add/components/LeaveRequestFormBuilder.tsx:80]: Understood, Sir. I'll focus on these areas next time.

File: src/common/FormTitle/FormTitle.tsx (hunk 3 above)
Comment (nadirhussain786) [reply src/common/FormTitle/FormTitle.tsx:5]: I have removed the duplicate classes.

File: src/common/MemberSelector/MemberSelector.tsx (hunk 1 above)
Comment (nadirhussain786) [reply src/common/MemberSelector/MemberSelector.tsx:91]: Fixed

**Review by ghulamhafeez — APPROVED** [review]

**Pull request merged**